hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/OzoneClientProducer.java

```diff
@@ -130,14 +139,29 @@ private OzoneClient getClient(OzoneConfiguration config)
     } catch (Throwable t) {
       // For any other critical errors during object creation throw Internal
       // error.
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("Error during Client Creation: ", t);
-      }
+      //      if (LOG.isDebugEnabled()) {
```

Comment:
Here the debug is removed, do we want to add the same when OS3Exception also?

Reply:
Good, question. I am thinking what supposed to be the right error handling.

AFAIK all of these exceptions are related to the user / input errors. They are not really application errors. User errors are not required to be logged, IMHO a permission denied is part of the normal business.

I would restore the `LOG.isDebugEnabled()` here, if you agree...  (8fe1202da)